modules/ROOT/pages/metrics-list.adoc

```diff
@@ -32,19 +32,20 @@ You can obtain them from applications, the Open Liberty runtime, and the Java vi
 They can be gathered and stored in database tools, such as https://prometheus.io/[Prometheus], and displayed on dashboards, such as https://grafana.com/[Grafana].
 For more information about building observability into your applications, see xref:microservice-observability-metrics.adoc[Microservice observability with metrics].
 
-When using MicroProfile Metrics with the feature:monitor-1.0[display=Performance Monitoring] feature, you enable the reporting of `vendor` metrics on the `/metrics` endpoint.
-Prior to MicroProfile Metrics 2.3, the feature:monitor-1.0[display=Performance Monitoring] feature needs to be explicitly configured.
-In MicroProfile Metrics 2.3 and up, the feature:monitor-1.0[display=Performance Monitoring] feature is automatically enabled by the MicroProfile Metrics feature during startup.
-It is worth noting that the _REST base metrics_ introduced in MicroProfile Metrics 2.3 also relies on the feature:monitor-1.0[display=Performance Monitoring] feature to function.
+To use the MicroProfile Metrics with the feature:monitor[display=Performance Monitoring] feature, enable the reporting of `vendor` metrics in the `/metrics` endpoint.
```

Comment:
This should be the other way around- when  you enable both the Performance Monitoring and MP Metrics features, you enable the reporting of vendor metrics on the metrics endpoint. 

Also, before this sentence, should add something like:
"When the MicroProfile metrics feature is enabled, a set of base metrics are always reported. You can augment this set of metrics by collecting vendor metrics, which are available from different monitoring components within your Open Liberty server.